pinot-common/src/main/java/org/apache/pinot/common/datablock/MetadataBlock.java

```diff
@@ -35,10 +34,12 @@
 public class MetadataBlock extends BaseDataBlock {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MetadataBlock.class);
-  @VisibleForTesting
-  static final int VERSION = 2;
   @Nullable
   private List<DataBuffer> _statsByStage;
+  private final int _stage;
+  private final int _worker;
```

Comment:
(minor) we usually call them `stageId` and `workerId`

Reply:
Fixed